components/board/analog_smoother.go

```diff
@@ -116,6 +117,10 @@ func (as *AnalogSmoother) Start(ctx context.Context) {
 				if errors.Is(err, errStopReading) {
 					break
 				}
+				// If the serial port was closed by the board, stop the analog smoother.
```

Comment:
The serial port refers to communications protocol - USB is one of the users of serial, in this day and age - it could refer to a usbA looking thing that speaks over RS-232 or RS-485, or a canbut looking thing that speaks over those two protocols as well.